tests/tls_apis/enc/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+include(oeedl_file)
+
+oeedl_file(../tls.edl enclave gen)
+
+add_executable(tls_enc enc.cpp ${gen})
+
+target_include_directories(tls_enc PRIVATE
+                          ${CMAKE_CURRENT_BINARY_DIR}
+                          ${PROJECT_SOURCE_DIR}/include/openenclave/corelibc)
```

Comment:
This is an internal test case, where a couple of internal oe_* api, defined in oecorelibc, were used. That's why corelibc path was included. 
In the upcoming external same code, it will solely base on the musl headers.